CODEOWNERS

```diff
@@ -18,3 +18,13 @@
 /cmd/cli/plugin/secret @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tkg-addons-owners
 /cmd/cli/plugin/pinniped-auth @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tkg-iam-owners
 /pkg/v1/tkg/tkgconfigpaths @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-release-team
+# API Machinery related files
+/pkg/v1/sdk @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-api-machinery-owners
+/cmd/managers/capabilities @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-api-machinery-owners
+/cmd/managers/featuregate @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-api-machinery-owners
+/cmd/cli/plugin-admin/codegen @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-api-machinery-owners
+/pkg/v1/codegen @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-api-machinery-owners
+/packages/management @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-api-machinery-owners
+/pkg/v1/providers/ytt/02_addons/management-packages @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-api-machinery-owners
+/hack/packages @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-api-machinery-owners
```

Comment:
Should we leave this to `@vmware-tanzu/tanzu-framework-reviewers` or add other teams (at least the addons team) since this is common for everyone?

Reply:
since these packages would be added/generated using the tooling we built I thought we should keep an eye on that dir for while at least until the process of adding the packages become solid.. we can continue to keep an eye or just add featuregates and capabilities dirs under our umbrella once the process becomes good enough.

Reply:
since these packages would be added/generated using the tooling we built I thought we should keep an eye on that dir for while at least until the process of adding the packages become solid.. we can continue to keep an eye or just add featuregates and capabilities dirs under our umbrella once the process becomes good enough.